### ISSUE TITLE
Create combined 2000-2010 tract weighting files for DC, MD, VA, WV

### ISSUE THREAD

**Issue by ptatian** [NeighborhoodInfoDC/General#25]
**>>> Before starting this issue, issues [3](https://github.com/NeighborhoodInfoDC/Census/issues/3) and [4](https://github.com/NeighborhoodInfoDC/Census/issues/4) in Census repo must be completed first.**

This task is to update the weighting data sets in the General library, **Wt_tr00_tr10**, **Wt_tr10_tr00**, **Wt_bg00_tr00**, **Wt_bg00_tr10**, **Wt_bg10_tr00**, **Wt_bg10_tr10** by adding tracts/block groups for MD, VA, and WV. 

Step 1: Modify the **D:\DCData\Libraries\General\Macros\Calc_weights_from_blocks.sas** program as follows.

- Code needs to be adapted to use the _%Finalize_data_set()_ macro. Change the _data_ step so that it saves to a temporary data set.

- Delete the _%File_info()_ macro call and replace with _%Finalize_data_set()_. Previously, we did the metadata registration in a separate program. Since it will now be handled here, we need to add a _revisions=_ parameter to the _%Calc_weights_from_blocks()_ macro.

Step 2: Modify the **D:\DCData\Libraries\General\Prog\Weights\Wt_all_2000_2010.sas** program as follows.

- Program uses old DCData setup. Need to replace the _%include_ statement with the current program template version.

- Program uses **Census.Blk_xwalk_2000_2010_dc**, **Census.Cen2000_sf1_dc_blks**, and **Census.Census_pl_2010_dc** data sets as inputs to create weighting files for DC. To create the combined weighting files, first create temporary versions of all three data sets that combine (concatenate) observations for DC, MD, VA, and WV into single files. These temporary data sets can then be used as inputs to the _%Calc_weights_from_blocks()_ macro. 

- Add a _revisions=_ parameter to all _%Calc_weights_from_blocks()_ calls so that the metadata will contain info on the changes to these files. Message should be something like, _%str(Add support for MD, VA, and WV tracts.)_  Note that enclosing the message with _%str()_ is necessary if the message contains commas, which otherwise would be interpreted as parameter delimiters. 

- The program has some testing code (_libname_, _proc sort_, _proc compare_) that is no longer relevant. These lines of code can be deleted.

**Charge code for this work: 932000-1601-000-00113**

**Comment by rpitingolo** [comment]
@ptatian thanks. I did a pull request for Census-#3 if you can review. It looks like you already did Census-#4?
